package.json

```diff
@@ -42,7 +42,7 @@
     "cross-env": "^7.0.3",
     "css-loader": "^7.1.2",
     "dotenv-webpack": "^8.1.0",
-    "eslint": "^8.57.1",
+    "eslint": "^9.16.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-prettier": "^5.2.1",
```

Comment:
Related to my other note about prettier, I suspect we just need the config dep and not the plugin itself?